src/components/pages/AdminTools/ManageUsers.js

```diff
@@ -76,9 +76,14 @@ const ManageUsersPage = props => {
     axiosWithAuth()
       .get(`/profile/${id}`)
       .then(res => {
-        console.log(res.data);
-        const { first_name, last_name, role, email, user_id } = res.data;
-        setFormValues({ first_name, last_name, role, email, user_id });
+        const { first_name, last_name, role_id, email, user_id } = res.data;
+        setFormValues({
+          first_name,
+          last_name,
+          role_id: role_id.toString(),

```

Comment:
Is there a reason role_id is being turned into a string instead of kept as an integer?